bosh/cloud_config.yml

```diff
@@ -100,6 +100,19 @@ networks:
       subnetwork_name: windows
       tags: [internal]
 
+- name: internal-wings
+  type: dynamic
+  subnets:
+  - az: z1
+    cloud_properties:
+      network_name: bosh
+      subnetwork_name: internal-wings
+      tags:
+      - internal
+      - wings-uaa-web
+      - wings-credhub-web
+
+
```

Comment:
can we use a separate config file / bosh configuration for that? it will be easier so we can update it without affecting the prod cloud config?